src/memote/utils.jl

```diff
@@ -106,4 +107,19 @@ to_element(x::String) = begin
     elements[sym]
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+A helper function that returns the median upper and lower bounds in a tuple. If none can be calculated
+(-1000.0, 1000.0) are set as the default values.
+"""
+function median_bounds(model::MetabolicModel)
+    lower_bound, upper_bound = bounds(model)
+    lb_list = [element for element in lower_bound if element != 0.0]
+    ub_list = [element for element in upper_bound if element != 0.0]
```

Comment:
bounds are usually on an exact float zero but it might be useful to work with `isapprox` here.